test/prob/wiener/wiener_test.hpp

```diff
@@ -65,6 +65,9 @@ class AgradDistributionWiener : public AgradDistributionTest {
     index.push_back(2U);
     value.push_back(numeric_limits<double>::infinity());
 
+    index.push_back(2U);
+    value.push_back(50.2);
+

```

Comment:
Is this the test for the new exception case?  It should have a unit test, not just a visual check.